### ISSUE TITLE
Error when running tutorial decompression step - view() takes from 0 to 2 positional arguments but 3 were given 

### ISSUE THREAD

**Issue by TamariMaayan** [baler-collaboration/baler#272]
**Describe the bug**
Ran the command from the tutorial:
poetry run python baler --project CFD_workspace CFD_project_animation --mode decompress

(After running tain and compress)

Received error:
 File "/baler/baler/modules/helper.py", line 580, in decompress
    decompressed = decompressed.view(
TypeError: view() takes from 0 to 2 positional arguments but 3 were given.

It did not save the decompressed data

![image](https://github.com/baler-collaboration/baler/assets/100471361/1378d953-9056-45b3-9520-0b53080101ea)

**Proposed Solution**
I could get it to run without error and produce the data file by applying the following change in helper.decompress()

Changed from:
if config.data_dimension == 2 and config.model_type == "dense":
        decompressed = decompressed.view(
            len(decompressed), number_of_columns, number_of_columns
        )

To:
if config.data_dimension == 2 and config.model_type == "dense":
        decompressed = **torch.from_numpy(**decompressed**)**.view(
            len(decompressed), number_of_columns, number_of_columns
        )

is that an acceptable solution?

**Comment by fritjof-b** [comment]
Hi,

I reproduced the error on my machine. I'm attaching a screenshot looking at the diff that (might have) introduced the error. Can you please advise @exook @singh96aman ? Tagging you because you worked on this last. Note that the current version is to the right. 

<img width="1233" alt="Screenshot 2023-06-20 at 10 42 48" src="https://github.com/baler-collaboration/baler/assets/60710890/aa1e1040-e2b1-4bee-84b1-8ee503987de9">

**Comment by singh96aman** [comment]
I'll take a look at this at the earliest!
We had to uncomment the code because the decoder needs to run in batches as well for large files. 

Even though I tested it, looks like we are not converting the numpy array to tensor

**Comment by exook** [comment]
@TamariMaayan try doing "len(decompressed), number_of_columns," on line 581 instead of "len(decompressed), number_of_columns, number_of_columns"

**Comment by singh96aman** [comment]
@exook Doing above doesn't fix the issue as it's necessary to convert a 60*2500 Tensor to 60*50*50 numpy array for plotting. 

The issue occurred because we uncommented the batched decoder part (which now directly returns a numpy array through detacher code). Earlier, there was an explicit line to convert it to numpy array.

`decompressed = decompressed.detach().cpu().numpy()`

@TamariMaayan resolution suggests to convert to Torch Tensor and then back to Numpy Array.
Alternatively, we could just use numpy reshape instead of Torch view to make it available for plotting.

I've made the change and raised a PR and tested and can see correct plots [Attached].

PR - https://github.com/baler-collaboration/baler/pull/275

[plotting.zip](https://github.com/baler-collaboration/baler/files/11828961/plotting.zip)
